cmd/virt-handler/virt-handler.go

```diff
@@ -146,6 +146,11 @@ func main() {
 	vmController.StartInformer(stop)
 	vmController.WaitForSync(stop)
 
+	err = configDiskClient.UndefineUnseen(vmStore)
```

Comment:
@rmohr  This should handle the startup condition you mentioned where stale data wasn't being cleaned up.